llvm/test/CodeGen/AMDGPU/GlobalISel/combine-fma-add-ext-mul.ll

```diff
@@ -73,10 +73,14 @@ define amdgpu_vs <5 x float> @test_5xf16_5xf32_add_ext_mul(<5 x half> inreg %x,
 ; GFX10-FAST-DENORM-NEXT:    v_mov_b32_e32 v2, s8
 ; GFX10-FAST-DENORM-NEXT:    v_mov_b32_e32 v3, s9
 ; GFX10-FAST-DENORM-NEXT:    v_mov_b32_e32 v4, s10
+; GFX10-FAST-DENORM-NEXT:    s_lshr_b32 s11, s0, 16
+; GFX10-FAST-DENORM-NEXT:    s_lshr_b32 s12, s1, 16
```

Comment:
This regressed gfx10 which has the more relaxed constant bus limit

Reply:
The `*v_fma_mix_f32` instruction still uses two SGPRs. The only issue here is, it can't see "through" the copies such that it can't directly use the source operands of the shifts via `op_sel`. I think the ISel version also has this limitation (apparently because here we "copied" the logic).

Reply:
Still should fix this in a follow up